### PR TITLE
add ability to scroll ios views

### PR DESCRIPTION
**Pull request by jlipps**
this works on ios7!

cc @dancuellar @dandoveralba

### CODE REVIEW

**Comment by xudafeng** [comment]
:+1:

**Comment by pavelzag** [comment]
Bumping up this discussion because https://github.com/appium/appium/blob/master/docs/gestures.md#scroll is broken
Are there any new for C# mobile:sweep implementation? Is it expected to be added in the upcoming versions?
Cheers, Pavel

File: docs/gestures.md [hunk 1]
```diff
@@ -222,6 +227,26 @@ In these examples, note that the element parameter is always optional.
...
+    // continue testing
+  });
+  ```
+
+* **Java:**
+
+  ```java
+  JavascriptExecutor js = (JavascriptExecutor) driver;
+  HashMap<String, Double> scrollObject = new HashMap<String, Double>();
+  scrollObject.put("direction", "down");
```
Comment (gustavjr) [inline docs/gestures.md]: Your HashMap is declared with < String, Double> and here your are passing < String, String> .

Comment (jlipps) [inline docs/gestures.md]: thanks, will fix that.

Comment (jlipps) [inline docs/gestures.md]: actually this is already fixed in master. this PR is ancient.

**Pull request merged**